code/game/turfs/simulated/walls_mineral.dm

```diff
@@ -13,6 +13,13 @@
 	walltype = "wood"
 	mineral = "wood"
 
+/turf/simulated/wall/mineral/brick
+	name = "brick wall"
+	desc = "A wall with brick siding, it looks nice"
+	icon_state = "Brick0"
```

Comment:
Yeah, tried placing the walls in build mode and the sprites werent working, is this why?